src/frontend/screens/WebView/index.tsx

```diff
@@ -104,7 +104,7 @@ export default function WebView() {
 
             setTimeout(() => {
               webview.findInPage('authorizationCode')
-            }, 500)
+            }, 50)
```

Comment:
Lowered the timeout here on purpose since it was not necessary to wait that much. Tested 4 or 5 times and the login is working fine.